README.md

```diff
@@ -6,114 +6,31 @@
 
-This is a Docker skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub Docker project
-started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit hooks](https://pre-commit.com)
-and [GitHub Actions](https://github.com/features/actions) configurations
-appropriate for Docker containers and the major languages that we use.
+This is a Docker project that containerizes the [li-pca-app](https://github.com/cisagov/li-pca-app)
+package, which can be used to run the Li-PCA Web Application project.
 
 ## Running ##
 
 ### Running with Docker ###
 
-To run the `cisagov/example` image via Docker:
+To run the `cisagov/li-pca-app-docker` image via Docker:
 
 ```console
-docker run cisagov/example:0.0.1
+docker run cisagov/li-pca-app-docker:0.0.1
 ```
 
 ### Running with Docker Compose ###
 
-1. Create a `docker-compose.yml` file similar to the one below to use [Docker Compose](https://docs.docker.com/compose/).
-
-    ```yaml
-    ---
-    version: "3.7"
-
-    services:
-      example:
-        image: cisagov/example:0.0.1
-        volumes:
-          - type: bind
-            source: <your_log_dir>
-            target: /var/log
-        environment:
-          - ECHO_MESSAGE="Hello from docker-compose"
-        ports:
-          - target: 8080
-            published: 8080
-            protocol: tcp
-    ```
-
 1. Start the container and detach:
 
     ```console
     docker-compose up --detach
     ```
 
-## Using secrets with your container ##
-
-This container also supports passing sensitive values via [Docker
-secrets](https://docs.docker.com/engine/swarm/secrets/).  Passing sensitive
-values like your credentials can be more secure using secrets than using
-environment variables.  See the
-[secrets](#secrets) section below for a table of all supported secret files.
-
-1. To use secrets, create a `quote.txt` file containing the values you want set:
-
-    ```text
-    Better lock it in your pocket.
-    ```
-
-1. Then add the secret to your `docker-compose.yml` file:
-
-    ```yaml
-    ---
-    version: "3.7"
-
-    secrets:
-      quote_txt:
-        file: quote.txt
-
-    services:
-      example:
-        image: cisagov/example:0.0.1
-        volumes:
-          - type: bind
-            source: <your_log_dir>
-            target: /var/log
-        environment:
-          - ECHO_MESSAGE="Hello from docker-compose"
-        ports:
-          - target: 8080
-            published: 8080
-            protocol: tcp
-        secrets:
-          - source: quote_txt
-            target: quote.txt
-    ```
-
-## Updating your container ##
-
-### Docker Compose ###
-
-1. Pull the new image from Docker Hub:
-
-    ```console
-    docker-compose pull
-    ```
-
-1. Recreate the running container by following the [previous instructions](#running-with-docker-compose):
-
-    ```console
-    docker-compose up --detach
-    ```
-
 ### Docker ###
```

Comment:
Why is all of this being removed? The last line is still setup to be a sub-heading of the `Updating your container` heading.